apps/frontend/src/App.tsx

```diff
@@ -18,11 +18,20 @@ import Page from './pages/Content';
 
 const ConfiguredLayout: React.FC = ({ children }) => {
   const { isAuthenticated } = useAuth0();
-  return isAuthenticated ? (
+  const user = useCurrentUser();
+  return isAuthenticated && user ? (
     <Layout
       libraryHref="/library"
       networkHref="/users"
       newsAndEventsHref="/news-and-events"
+      profileHref={`/users/${user.id}`}
+      teams={[]} // TODO
+      settingsHref="/settings"
+      feedbackHref="/feedback"
+      logoutHref="/logout" // TODO
```

Comment:
This adds (frontend) and removes (LoginLogoutButton) a lot of code that would make the PR too big, doing it in a follow up